WPMedia/fusion-news-theme-blocks
# add logical properties to right-rail-block, right-rail-advanced-block…

…, and single-column-layout-block plus add styles to storybook

## Jira Ticket

- [THEMES-1223][(https://arcpublishing.atlassian.net/browse/THEMES-1223)](https://arcpublishing.atlassian.net/browse/THEMES-1223)

## Acceptance Criteria

    Update padding, margin, and border properties. The following conversions must be done:

        *-top and *-bottom need to be changed to *-block.

        *-left and *-right need to be changed to *-inline.

    height and width will need to be set to block-size and inline-size respectively

Tested on: 

    right-rail-block

    right-rail-advanced-block

    single-column-block 

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout logical-properties-layout`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/right-rail-block,@wpmedia/right-rail-advanced-block,@wpmedia/single-column-layout-block`
3. Set "textDirection": "rtl" in the "siteProperties" object of the blocks.json
4. Check pages in pagebuilder using the 3 layouts. Right rail layouts will have the rail on the left. [should we rename those as just rail-layouts???]


Storybook has been blindly updated as I cannot get it to run. Single column layout did not have a story and I didn't want to add one without being able to actually run it/see it

[THEMES-1223]: https://arcpublishing.atlassian.net/browse/THEMES-1223?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
